docs/source/ddpo_trainer.mdx

```diff
@@ -0,0 +1,103 @@
+# Credits
```

Comment:
Can you only add the title with a top level section (`#`) with the title being `Denoising Diffusion Policy Optimization` and then adjusting the subsections accordingly? I have a slight preference to put credits/references at the end of the document but no strong opinion.